app/views/shared/_news_signup_inpage.html.erb

```diff
@@ -0,0 +1,70 @@
+<div class="news-signup-test
```

Comment:
the -test stuff is from the older version. This may change before the WIP is removed however we are A/B Testing the old versus new so need to work out a nice way of showing one or the other.